c2corg_api/models/route.py

```diff
@@ -322,3 +322,7 @@ class ArchiveRouteLocale(_RouteLocaleMixin, ArchiveDocumentLocale):
     'locales.title', 'locales.title_prefix', 'elevation_min', 'elevation_max',
     'activities'
 ])
+schema_association_waypoint_route = restrict_schema(schema_route, [
+    'locales.title', 'locales.title_prefix', 'elevation_min', 'elevation_max',
+    'activities', 'geometry.geom_detail'
```

Comment:
Loading the simple geom (point) for associated routes is useless: the cool thing is to display the tracks of the associated routes in the map of a WP detail view.
But you're right: not sure what will happen if there are too many routes with a ``geom_detail``. I think it's worth trying.